package.json

```diff
@@ -35,8 +34,7 @@
     "watch:sass": "npm run css-compile -- --watch"
   },
   "peerDependencies": {
-    "bootstrap": "5.2.2",
-    "popper.js": "1.16.1"
```

Comment:
This popper version is incorrect - bootstrap@5 uses popper@2, and has its own peer dependency for it.